analyzer/spec/functional/surface_spec_spec.rb

```diff
@@ -7,48 +7,77 @@ module Interpreter
       let(:concept) { Concepts::SurfaceSpec.new(:spec_name) }
       let(:spec) { Interpreter::SurfaceSpec.new(concept) }
 
-      before(:each) do
+      before do
         interpret_basis
+        spec.interpret('atoms c1: C%d, c2: C%d, c3: C%d')
       end
 
-      describe '#position' do
-        describe 'both atoms has lattice' do
-          let(:incomplete) { syntax_error('position.incomplete') }
+      describe 'default behavior for position instances' do
+        it { expect {
+            spec.interpret("position :c1, :c2, face: 100, dir: :front")
+          }.not_to raise_error }
 
-          before(:each) do
-            spec.interpret('atoms c1: C%d, c2: C%d')
-          end
-
-          it { expect { spec.interpret('position :c1, :c2, face: 100') }.
-            to raise_error(*incomplete) }
+        it { expect {
+            spec.interpret("no-position :c1, :c2, face: 100, dir: :front")
+          }.to raise_error(*syntax_error('non_position.impossible')) }
```

Comment:
`end` at 22, 10 is not aligned with `expect {` at 20, 13 or `it { expect {` at 20, 8